magkit-test-cms/pom.xml

```diff
@@ -0,0 +1,121 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>de.ibmix.magkit</groupId>
+        <artifactId>magkit-test</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>magkit-test-cms</artifactId>
+    <name>magkit-test-cms Magnolia Module</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <version.magkit-test-jcr>1.0.0-SNAPSHOT</version.magkit-test-jcr>
+        <version.magkit-test-servlet>1.0.0-SNAPSHOT</version.magkit-test-servlet>
+        <version.mockito-core>1.10.19</version.mockito-core>
+        <version.magnolia-core>6.2.17</version.magnolia-core>
```

Comment:
Use version.magnolia property instead of new property of core module.

Reply:
If this to force downgrade of tests-Library, then we have to fix this later by upgrading used Mockito version.